theme/cobalt2.json

```diff
@@ -294,10 +296,7 @@
   "tokenColors": [
     {
       "name": "Comment",
-      "scope": [
-        "comment",
-        "punctuation.definition.comment"
-      ],
```

Comment:
@wesbos, I just noticed all these formatting diffs. LMK if you'd like me to push another commit to slim down the diff just to the inlay hint updates.